generator/README.md

```diff
@@ -24,6 +24,19 @@ The generator reads in from `generator.yml` and writes to `snmp.yml`.
 
 Additional command are available for debugging, use the `help` command to see them.
 
+## Docker Users
+
+If you would like to run the generator in docker to generate your `snmp.yml` config run the following commands.
+
+```
+docker build -t snmp-generator .
+docker run -ti \
+  -v $PWD/mibs:/root/.snmp/mibs \
```

Comment:
Usually mibs would be in $HOME/.snmp/mibs

Reply:
/root is $HOME in this container

Reply:
I meant on the LHS

Reply:
Oh I see - For my use case I'm pulling down vendor specific mibs for network devices - so there is no canonical mib location

I'm not opposed to changing this to $HOME/.snmp/mibs if that works better for you

Reply:
That's where net-snmp will usually be looking, so where I'd expect users to have them.

Reply:
done